tox.ini

```diff
@@ -1,21 +1,26 @@
 [tox]
 minversion=2.3.1
 envlist =
-    py{27,35}-ansible22-{pylint,unit,flake8,yamllint,generate_validation}
+    py{27,35}-{lint,unit}
 skipsdist=True
 skip_missing_interpreters=True
 
 [testenv]
 deps =
+    -rrequirements.txt
```

Comment:
Do we really expect many users to come in and try to run and test manually? Tox already treats the deps list as a single pip install.